src/EnergyPlus/DataHeatBalSurface.hh

```diff
@@ -133,8 +133,10 @@ struct HeatBalSurfData : BaseGlobalStruct
     Array1D<Real64> SurfQdotRadSolarInRep;        // Surface thermal radiation heat transfer inside face surface [W]
     Array1D<Real64> SurfQdotRadSolarInRepPerArea; // [W/m2]Surface thermal radiation heat transfer rate per m2 at Inside face surf
     // these next two all are for Lights visible radiation gains on inside face
-    Array1D<Real64> SurfQRadLightsInReport; // Surface thermal radiation heat gain at Inside face [J]
-    Array1D<Real64> SurfQdotRadLightsInRep; // Surface thermal radiation heat transfer inside face surface [W]
+    Array1D<Real64> SurfQRadLightsInReport;           // Surface thermal radiation heat gain at Inside face [J]
```

Comment:
Why are some of these `InRep` and others `InReport`?

Reply:
@amirroth Not sure there is a specific reason.  I will look at cleaning that up in the next commit.